src/help/command.ts

```diff
@@ -0,0 +1,58 @@
+import CommandHelp from '@oclif/plugin-help/lib/command';
+import * as Config from '@oclif/config';
+import {renderList} from '@oclif/plugin-help/lib/list';
+import chalk from 'chalk';
+import indent from 'indent-string';
+
+const {
+  underline,
+  bold,
+} = chalk;
+const {
+  dim,
+} = chalk;
```

Comment:
```suggestion
const {
  underline,
  bold,
  dim,
} = chalk;
```

Reply:
I have moved the help command now it does not show in the root help message.